aws-cpp-sdk-core/CMakeLists.txt

```diff
@@ -402,6 +397,13 @@ else()
     endif()
 endif()
 
+configure_file("${CMAKE_SOURCE_DIR}/toolchains/pkg-config.pc.in" "${PROJECT_NAME}.pc" @ONLY)
```

Comment:
Why was this moved here?  Core's config and install  is now slightly different than all the other libraries.